lms/djangoapps/teams/static/teams/templates/edit-team.underscore

```diff
@@ -48,9 +80,10 @@
     <button class="action action-cancel">
         <%=
             interpolate_text(
-                _.escape(gettext("Cancel {span_start}a new team{span_end}")),
+                _.escape(gettext("Cancel {span_start} {action} team {span_end}")),
                 {
-                    'span_start': '<span class="sr">', 'span_end': '</span>'
+                    'span_start': '<span class="sr">', 'span_end': '</span>',
+                    'action': action === 'create' ? 'creating' : 'updating'
```

Comment:
i18n ("creating", "updating").

This whole thing is going to very difficult for a translator to deal with. I recommend you split it up into 2 different strings (one for the updating case, one for creating), and also add translator comments. The concatenation approach here with individual words being substituted in won't work.